xml/templates/message_templates/participant_cancelled_text.tpl

```diff
@@ -7,10 +7,10 @@
 {ts}Event Information and Location{/ts}
 
 ===========================================================
-{$event.event_title}
-{$event.event_start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|date_format:"%Y%m%d" == $event.event_start_date|date_format:"%Y%m%d"}{$event.event_end_date|crmDate:0:1}{else}{$event.event_end_date|crmDate}{/if}{/if}
+{$event.title}
```

Comment:
@eileenmcnaughton just noticed that you have made this change `event_title` to just `title` but not in L37 of the html version of the change

Reply:
ug - that's wrong - re-fixing